twenty-first/src/util_types/merkle_tree.rs

```diff
@@ -689,6 +706,23 @@ pub mod merkle_tree_test {
         assert_eq!(MerkleTreeError::IncorrectNumberOfLeafs, err);
     }
 
+    #[proptest]
+    fn merkle_tree_makers_behave_identically_on_random_input(leafs: Vec<Digest>) {
+        let parallel = MerkleTree::new::<CpuParallel>(&leafs);
+        let sequential = MerkleTree::new::<CpuSequential>(&leafs);
+        prop_assert_eq!(parallel, sequential);
+    }
+
+    #[proptest]
```

Comment:
cases here, could be limited to 10 or 20, I think.

Reply:
The tests finishes in less than 2 seconds on my laptop. From this perspective, I don't think the current number of cases is a problem, but maybe there are other considerations to take into account?

Reply:
If it's that fast, don't bother changing anything.